build-packages.cmd

```diff
@@ -2,61 +2,43 @@
 setlocal EnableDelayedExpansion
 
 set "__ProjectDir=%~dp0"
-set packagesLog=build-packages.log
-set binclashLoggerDll=%~dp0Tools\net45\Microsoft.DotNet.Build.Tasks.dll
-set binclashlog=%~dp0binclash.log
-echo Running build-packages.cmd %* > %packagesLog%
-
-set options=/nologo /maxcpucount /v:minimal /clp:Summary /nodeReuse:false /flp:v=detailed;Append;LogFile=%packagesLog% /l:BinClashLogger,%binclashLoggerDll%;LogFile=%binclashlog% /p:FilterToOSGroup=Windows_NT
 set allargs=%*
 
 if /I [%1] == [/?] goto Usage
 if /I [%1] == [/help] goto Usage
 
-REM ensure that msbuild is available
-echo Running init-tools.cmd
-call %~dp0init-tools.cmd
-
-set __msbuildArgs="%__ProjectDir%\src\.nuget\Microsoft.NETCore.Runtime.CoreClr\Microsoft.NETCore.Runtime.CoreCLR.builds" !allargs!
-echo msbuild.exe %__msbuildArgs% !options! >> %packagesLog%
-call msbuild.exe %__msbuildArgs% !options!
+echo %__ProjectDir%/run.cmd build-packages -Project=%__ProjectDir%/src/.nuget/Microsoft.NETCore.Runtime.CoreClr/Microsoft.NETCore.Runtime.CoreCLR.builds -FilterToOSGroup=Windows_NT %allargs%
+call %__ProjectDir%/run.cmd build-packages -Project=%__ProjectDir%/src/.nuget/Microsoft.NETCore.Runtime.CoreClr/Microsoft.NETCore.Runtime.CoreCLR.builds -FilterToOSGroup=Windows_NT %allargs%
 if NOT [!ERRORLEVEL!]==[0] (
-  echo ERROR: An error occurred while building packages, see %packagesLog% for more details.
+  echo ERROR: An error occurred while building CoreCLR Runtime package, see build-packages.log for more details.
   exit /b 1
 )
 
-set __msbuildArgs="%__ProjectDir%\src\.nuget\Microsoft.NETCore.Jit\Microsoft.NETCore.Jit.builds" !allargs!
-echo msbuild.exe %__msbuildArgs% !options! >> %packagesLog%
-call msbuild.exe %__msbuildArgs% !options!
+echo %__ProjectDir%/run.cmd build-packages -Project=%__ProjectDir%/src/.nuget/Microsoft.NETCore.Jit/Microsoft.NETCore.Jit.builds -FilterToOSGroup=Windows_NT %allargs%
+call %__ProjectDir%/run.cmd build-packages -Project=%__ProjectDir%/src/.nuget/Microsoft.NETCore.Jit/Microsoft.NETCore.Jit.builds -FilterToOSGroup=Windows_NT %allargs%
```

Comment:
You could create an alias for `FilterToOSGroup` and then pass default. This is going to run in windows so run.exe should set the value to `Windows_NT` and that way you don't need to sent it in every call.